arwen/contexts/runtime.go

```diff
@@ -684,6 +684,12 @@ func (context *runtimeContext) VerifyContractCode() error {
 			logRuntime.Trace("verify contract code", "error", err)
 			return err
 		}
+	} else {
```

Comment:
not with else - be specific. add another if separatly

Reply:
fixed